interact.js

```diff
@@ -4245,6 +4276,14 @@
                     this.options.resize.square = options.square;
                 }
 
+                if (isBool(options.preserveAspectRatio)) {
+                    this.options.resize.preserveAspectRatio = options.preserveAspectRatio;
+                }
+
+                if (this.options.resize.square && this.options.resize.preserveAspectRatio) {
+                    throw new Error('resizable.square and resizable.preserveAspectRatio cannot be used together');
```

Comment:
I prefer to avoid throwing custom errors. It would be better to simply give `preserveAspectRatio` preference by checking for it before `square` at [line 2007](https://github.com/taye/interact.js/pull/260/files#diff-a3d8d17fae1b42124a71fe9aa8b26d6fR2007).